blinkit-app/src/homePage/atoms/InsertImg.js

```diff
@@ -0,0 +1,10 @@
+import React, { Component } from 'react'
+ class InsertImg extends Component {
+  render() {
+    return (
+     <img src={this.props.src} alt={this.props.alt} />
```

Comment:
This component is not required simply img tag would be sufficient unless we need any extra styling or divs here